frontend/public/components/app-contents.tsx

```diff
@@ -291,6 +291,17 @@ const AppContents: React.FC<{}> = () => {
                   />
                 )}
               />
+              <Route
```

Comment:
To make things easier in the future to really see what routes were impacted, I left the new route next to the old route instead of creating a helper.